sample/app-ios/app-ios/KittenView.swift

```diff
@@ -0,0 +1,86 @@
+import SwiftUI
+import Shared
+
+struct KittenView: View {
+    
+    private let component: KittenComponent
+    private let kittenSize: KittenSize
+
+    @ObservedObject
+    private var model: ObservableValue<KittenComponentModel>
+
+    private var imageName: String {
+        switch model.value.imageType {
+        case .cat1: return "cat1"
+        case .cat2: return "cat2"
+        case .cat3: return "cat3"
+        case .cat4: return "cat4"
+        case .cat5: return "cat5"
+        default: return ""
+        }
+    }
+
+    private var fontSize: CGFloat {
+        switch kittenSize {
+        case .large: return 16
+        case .small: return 10
+        }
+    }
+    
+    init(_ component: KittenComponent, _ kittenSize: KittenSize) {
+        self.component = component
+        self.kittenSize = kittenSize
+        model = ObservableValue(component.model)
+    }
+
+    var body: some View {
+        ZStack {
+            Image(imageName)
+                    .resizable()
+                    .aspectRatio(contentMode: .fill)
+            VStack {
+                Text(model.value.text)
+                        .padding(8)
+                        .frame(maxWidth: .infinity)
+                        .background(LinearGradient(
+                                colors: [.black.opacity(0.75), .clear],
+                                startPoint: .top,
+                                endPoint: .bottom
+                        ))
+                        .foregroundColor(.white)
+                        .modifier(AnimatableFontModifier(size: fontSize))
+                Spacer()
+            }
+        }
+    }
+}
+
+extension KittenView {
```

Comment:
The same question here. Maybe put the content as top-level and private?

Reply:
Agree, made them private and top-level